plugins/powerup/src/main/java/edu/wpi/first/shuffleboard/plugin/powerup/PowerUpPluginVersion.java

```diff
@@ -0,0 +1,18 @@
+package edu.wpi.first.shuffleboard.plugin.powerup;
+
+/**
+ * Autogenerated version class containing the version of the PowerUp plugin.
+ *
+ * <p><strong>DO NOT MODIFY THIS CLASS</strong>
+ */
+public final class PowerUpPluginVersion {
+
+  /**
+   * The version of the PowerUp plugin.
+   */
+  public static final String VERSION = "1.5.0";
+
+  private PowerUpPluginVersion() {
+    throw new UnsupportedOperationException();
+  }
+}
```

Comment:
EOL @ EOF

Reply:
You know spotless can enforce this right?